CHANGELOG.rst

```diff
@@ -22,6 +22,19 @@ Retro-compatibility breaks
 
 - Add nfd test_master_branch protocol `#179 https://github.com/openshift-psap/ci-artifacts/pull/179>`_
 
+Bug fixes
+~~~~~~~~~
+
+- Use ``subscriptions.operators.coreos.com`` instead of
+  ``subscriptions`` to avoid conflicts with KNative `subscriptions
+  <https://knative.dev/docs/eventing/channels/subscriptions/>`_ `#207
```

Comment:
A super-nit, *but*, I'd try to be consistent with `/` URL terminations.  If not in the entire document, then at least within a PR.  I.e., decide whether to keep them or drop them.